cli/coverage_cmd.go

```diff
@@ -70,18 +73,32 @@ func (c *CoverageCmd) Execute(args []string) error {
 	return nil
 }
 
-var codeExts = []string{".go", ".java", ".py", ".rb", ".cpp", ".ts", ".cs", ".js", ".php", ".m"} // codeExt lists all file extensions that indicate a code file we want to cover
-var codeExts_ = make(map[string]struct{})
+var langToExts = map[string][]string{
+	"Go":          []string{".go"},
+	"Java":        []string{".java"},
+	"Python":      []string{".py"},
+	"Ruby":        []string{".rb"},
+	"C++":         []string{".cpp"},
+	"TypeScript":  []string{".ts"},
+	"C#":          []string{".cs"},
+	"JavaScript":  []string{".js"},
+	"PHP":         []string{".php"},
+	"Objective-C": []string{".m"},
+}
+var extToLang map[string]string
```

Comment:
What if there are multiple file extensions denoting the same language (example: .c++, .cxx, .cpp, .cc)?